packages/api/src/external/carequality/command/cq-directory/get-organizations-for-xcpd.ts

```diff
@@ -14,6 +15,16 @@ function sortOrgsByNearbyOrder(orgs: CQDirectoryEntryModel[], orderMap: Map<stri
   });
 }
 
+export async function getAllCQOrgsIds(): Promise<Set<string>> {
+  const orgs = await Promise.all([getOrganizationsWithXCPD()]);
```

Comment:
Why not just `await getOrganizationsWithXCPD()`?

We should be very careful with `Promise.all`, it can use all resources we have (imaging doing a single DB query for every item of a list of 1,500 promises).